src/Ledger/Prelude.agda

```diff
@@ -86,7 +86,7 @@ open import Axiom.Set.Map th public
   renaming (Map to _⇀_)
 
 open import Axiom.Set.TotalMap th public
-open import Axiom.Set.TotalMapOn th public
+open import Axiom.Set.TotalMapOn th
```

Comment:
This line was inserted just to make sure the build tests don't fail with error "TotalMapOn.agdai" doesn't exist.  (But it need not be inherited by modules that import this one, so no need for `public`.)